src/notification.c

```diff
@@ -68,8 +68,9 @@ void notification_print(const struct notification *n)
         printf("\tformatted: '%s'\n", n->msg);
         printf("\tfg: %s\n", n->colors.fg);
         printf("\tbg: %s\n", n->colors.bg);
-        printf("\thighlight: %s\n", n->colors.highlight);
-        printf("\tframe: %s\n", n->colors.frame);
+        printf("\thighlight: ");
+        for (int i = 0; n->colors.highlight[i]; i++) printf("%s ", n->colors.highlight[i]);
```

Comment:
Nit pick: this would leave a trailing white space and moving the line break to the following entry seems a bit fragile (when lines are moved around). How about something like

```c
gchar *highlight = g_strjoinv(" ", n->colors.highlight);
printf("\thighlight: %s\n", highlight);
g_free(highlight);
```

Reply:
Fair enough, I wasn't satisfied with that in the first place